Unity/back button menu and speech recognition for menus

anreeta opened the pull request:
* Add back button to menus
* Add are you ready screens
* Add speech recognition for all menus

Please check menus work and nothing is now broken

rachel92124 commented:
"Resume" pulls up pause menu and I assume it's supposed to resume the game instead. It seems like the voice commands for the level selection screen aren't working for me. I think you forgot to attach the script to the levelSelector canvas. Also, do we want to change the quit to exit on the pause menu since the voice command is "exit"?

anreeta commented:
I should have fixed the following issues:
- "Resume" pulls up pause menu thing
- voice controls for level selection screen 

I have added extra changes as well:
- Updated readme
- both "quit" and "exit" are valid voice commands on the pause menu
- changed some words on the Main Menu to reflect voice commands
- voiced Start Screen
- voiced End Screen

Thank you for taking a look
